.github/workflows/check-labels.yml

```diff
@@ -1,7 +1,7 @@
 name: Check Labels
 
 on:
-  pull_request:
+  pull_request_target:
     types: [opened, synchronize, reopened, labeled, unlabeled]
```

Comment:
```suggestion
  pull_request:
    types: [opened, synchronize, reopened, labeled, unlabeled]
    branches-ignore: [main]
  pull_request_target:
    types: [opened, synchronize, reopened, labeled, unlabeled]
    branches: [main]
```

Reply:
For something bad to happen with non-main branch, the attacker needs to have write permissions to the repo.
And with the write permissions, anything can be changes.
But I'm fine with this change if it feels safer.

Reply:
Discussed offline, will do the suggested way.